text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,161 @@
-Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
 
+ .d8888b.
+d88P  Y88b
+888    888
+888         .d88b.  88888b.  88888b.   .d88b.  888d888
+888        d88""88b 888 "88b 888 "88b d88""88b 888P"
+888    888 888  888 888  888 888  888 888  888 888
+Y88b  d88P Y88..88P 888  888 888  888 Y88..88P 88
+ "Y8888P"   "Y88P"  888  888 888  888  "Y88P"  888
+
+Version 1.0.0
+__________________________________________________________________
+Hi, my name is Connor! I'm your personalised android assistant.
+Loading your current tasks...
+__________________________________________________________________
+Here are your current tasks: 
+    1. [T][ ] Do Laundry
+    2. [D][ ] Study (By: 9pm)
+    3. [E][X] Birthday Party (At: Saturday Afternoon)
+    4. [E][ ] Meet Friend (At: Sunday 12pm)
+__________________________________________________________________
+>>> clear
+__________________________________________________________________
+Poof! All your tasks are cleared!
+__________________________________________________________________
```

Comment:
Just a usability thing, it would be good to ask the user for confirmation before this potentially irreversible task is carried out to prevent the user from losing progress :)